aws/ci_build

```diff
@@ -144,11 +144,31 @@ Showing last #{CHAT_LOG_TAIL_LENGTH} lines of build log:
       ChatClient.message 'server operations', message, color: 'red', notify: 1
       ChatClient.message 'server operations', commit_url, color: 'gray', message_format: 'text'
     end
+
+    emit_deploy_metric(rack_env.to_s.titleize, status == 0)
   end
 
   # Return the same output and status code that BUILD returned.
   puts log
   status
 end
 
+def emit_deploy_metric(stage, successful)
+  cw = Aws::CloudWatch::Client.new
+
+  if cw
```

Comment:
I don't think this conditional is necessary- if the instance creation fails it will raise its own error, I don't think it will ever return `nil` for an object.